### PR TITLE
Re scss report

### PR DESCRIPTION
**Pull request by aamesaa**
This PR is about rename the class in Report page using BEM standart. In this pull request, the download button is worked, it can download the table in HTML into a .csv file

### CODE REVIEW

**Review by sebastianbagya — APPROVED** [review]
OK

File: src/component/Report.vue [hunk 1]
```diff
@@ -1,27 +1,259 @@
...
         },
-        components:{Filters, ReportTable}
+
+        methods: {
+            showReport: function() {
+                this.$http.get('http://localhost:8080/report', {params:{
+                    dept:this.selected,
+                    startDate:this.mulai,
+                    endDate:this.selesai
+                }})
```
Comment (riencus) [inline src/component/Report.vue]: `this.mulai` & `this.selesai`? Where does it come from? I don't see any assignment to both of those field

File: src/component/Report.vue [hunk 2]
```diff
@@ -1,27 +1,259 @@
...
+                    this.reportData = response.data;
+                })
+                .catch(function (error) {
+                  console.log(error);
+                })
+            },
+
+            download_csv: function(csv, filename) {
+              var csvFile;
+              var downloadLink;
```
Comment (riencus) [inline src/component/Report.vue:113]: I just want to point up the fact that there is an inconsistency on the method naming style. Please change it to `camelCase`

**Review by riencus — CHANGES_REQUESTED** [review]

File: src/component/Report.vue (hunk 1 above)
Comment (aamesaa) [reply src/component/Report.vue]: I refactored the `v-model` into `startDate` and `endDate`. Already fix it

File: src/component/Report.vue [hunk 3]
```diff
@@ -134,7 +134,7 @@
               downloadLink.click();
           },
 
-          export_table_to_csv: function(html, filename) {
+          export_table_to_csv: function(html, fileName) {
           	var csv = [];
```
Comment (riencus) [inline src/component/Report.vue]: You forgot to `camelCase` this one

**Review by riencus — CHANGES_REQUESTED** [review]

**Comment by aamesaa** [comment]
All requested changed have been committed

**Review by riencus — APPROVED** [review]

**Pull request merged**